action-templates/actions/action-build-docs/action.yml

```diff
@@ -1,27 +1,23 @@
 # https://vitepress.dev/guide/deploy#github-pages
-name: Build docs
+name: "Build docs"
 
 inputs:
   docs-path:
+    description: "Path to vitepress docs project"
     required: false
     default: "./docs"
-    type: string
```

Comment:
das finde ich auch vom Compiler bisschen irreführend, dass er da keinen Fehler wirft, wenn man ein Attribute zu viel hinzufügt. Kann man irgendwo ein Bug Ticket bei Github machen? @it-at-m/itm-admin